storm-core/src/jvm/org/apache/storm/ui/exceptionmappers/NotAliveExceptionMapper.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.storm.ui.exceptionmappers;
+
+import org.apache.storm.generated.NotAliveException;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class NotAliveExceptionMapper implements ExceptionMapper<NotAliveException> {
+    public Response toResponse(NotAliveException ex) {
+        return Response.status(Response.Status.INTERNAL_SERVER_ERROR).build();
```

Comment:
Could you file a follow on JIRA to handle this better in both the response and in the UI?

Reply:
I'd rather not accept unless the response is at least similar detail on current. (This also means current response is not a good format IMHO, and I don't want to make it worse.)